typhos/plugins/core.py

```diff
@@ -83,6 +83,8 @@ def __init__(self, channel, address, protocol=None, parent=None):
         self._connection_open = True
         self.signal_type = None
         self.is_float = False
+        self.enum_strs = ()
```

Comment:
I got confused for a bit about what type this actually ends up as, it's not always a tuple right?  Maybe a type hint here would help, since it's buried in `send_new_meta`?

Reply:
Let me check, I think it's always a tuple but it might also be a list

Reply:
It looks like `enum_strs` is always casted to a tuple when we receive them from the control layer before filling the `metadata` dictionary, and are again casted to a tuple in `describe`. I'll annotate this as a tuple.